lib/workers/branch/index.ts

```diff
@@ -345,6 +345,13 @@ export async function processBranch(
     ) {
       logger.debug('Manual rebase requested via Dependency Dashboard');
       config.reuseExistingBranch = false;
+    } else if (branchExists && config.rebaseWhen === 'never') {
```

Comment:
So you are no longer able to trigger a rebase by using the checkbox?

Reply:
Shouldn't it still work due to the `if` above?

Reply:
`config.rebaseRequested` isn't checked there. 🤔

Reply:
I'll check this manually